app/models/custom_button.rb

```diff
@@ -139,7 +139,7 @@ def self.available_for_user(user, group)
   end
 
   def self.get_user(user)
-    user = User.in_region.find_by_userid(user) if user.kind_of?(String)
+    user = User.find_by_userid(user) if user.kind_of?(String)
```

Comment:
Right, this is little bit confusing. The key is to look into User class -> it does override `find_by_userid` with in_region scope.

Instead of using the scope manually and then calling `find_by_userid` on it, we just use custom `find_by_userid` defined by User.